src/NServiceBus.Core.Tests/Routing/Routers/UnicastSendRouterConnectorTests.cs

```diff
@@ -166,14 +166,14 @@ public class UnicastSendRouterConnectorTests
         [Test]
         public async Task Should_route_using_the_mappings_if_no_destination_is_set()
         {
-            var strategy = new FakeRoutingStrategy
+            var strategy = new FakeSendRouter
```

Comment:
nitpick: `strategy` should probably be renamed to `router` or something. Same for the other tests in this file.